app/assets/stylesheets/components/_result.scss

```diff
@@ -1,149 +1,44 @@
 .result {
   @include clearfix();
-  border: 1px solid $result-border-color;
-  border-radius: $result-radius;
-  margin-bottom: $baseline-unit*6;
 
-  .svg-icon {
-    @include inline-block();
-    position: relative;
-    top: 5px;
-    margin-right: $baseline-unit;
-  }
+  margin-bottom: $baseline-unit*6;
+  padding: $baseline-unit;
 
-  .svg-icon-email,
-  .svg-icon-url,
-  .svg-icon-phone {
-    width: 20px;
-    height: 20px;
-    fill: $color-green-secondary;
-  }
+  border: 1px solid $result-border-color;
+  border-radius: $result-radius;
+  -webkit-box-shadow: 0 1px 0 0 $result-shadow-color;
+  -moz-box-shadow: 0 1px 0 0 $result-shadow-color;
```

Comment:
The result design has a darker font for the text. I think what you want is something like:
```css
/* _result.scss */
.result {
  color: $result-text-color;
}

/* settings.scss */
$result-text-color: $color-black;
```